js/viz/pie_chart.d.ts

```diff
@@ -45,21 +45,32 @@ import {
 } from './core/base_widget';
 
 import {
-    VizPalette,
-    VizWordWrap,
-    VizTextOverflow,
+    ChartsDataType,
     DashStyle,
-    ChartDataType,
-    PieChartLegendHoverMode,
-    PieChartResolveLabelOverlapping,
-    PieChartType,
-    PieChartSegmentsDirection,
-    PieChartAnnotationLocation,
     HatchingDirection,
-    SmallValuesGroupingMode,
-    PieChartSeriesInteractionMode,
-    PieChartLabelPosition,
-} from '../types/enums';
+    LabelPosition,
+    Palette,
+    TextOverflow,
+    WordWrap,
+} from '../common/charts';
+
+export {
+    ChartsDataType,
+    DashStyle,
+    HatchingDirection,
+    LabelPosition,
+    Palette,
+    TextOverflow,
+    WordWrap,
+};
+
+export type PieChartAnnotationLocation = 'center' | 'edge';
+export type PieChartLegendHoverMode = 'none' | 'allArgumentPoints';
+export type PieChartResolveLabelOverlapping = 'hide' | 'none' | 'shift';
```

Comment:
```suggestion
export type PieChartResolveLabelOverlapMode = 'hide' | 'none' | 'shift';
```

Reply:
Renamed it to `PieChartLabelOverlap` to keep consistent with others of the kind